### ISSUE TITLE
Error installing 1.1.1

### ISSUE THREAD

**Issue by darkedges** [ConsumerDataStandardsAustralia/java-artefacts#102]
```
git checkout tags/cds-java-artefacts-1.1.1
mvn install -Ddockerfile.skip=true
```

returns
```
8 errors
3 warnings
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for CDS Java Artefacts 1.1.1:
[INFO]
[INFO] CDS Java Artefacts ................................. SUCCESS [  0.746 s]
[INFO] API Model .......................................... SUCCESS [02:09 min]
[INFO] Reflection ......................................... SUCCESS [ 15.912 s]
[INFO] Code Generator ..................................... SUCCESS [ 43.168 s]
[INFO] Client Library ..................................... FAILURE [ 49.449 s]
[INFO] Reference Test ..................................... SKIPPED
[INFO] Client CLI ......................................... SKIPPED
[INFO] Data Holder ........................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:02 min
[INFO] Finished at: 2020-02-17T21:13:28Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project client: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\api\BankingProductsAPI.java:325: warning: no @param for version
[ERROR]     public okhttp3.Call listProductsCall(ParamEffective effective, OffsetDateTime updatedSince, String brand, ParamProductCategory productCategory, Integer version, Integer page, Integer pageSize, final ApiCallback _callback) throws ApiException {
[ERROR]                         ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceIntervalSchedule.java:58: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceIntervalSchedule.java:58: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                      ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceIntervalSchedule.java:58: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                                                                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceIntervalSchedule.java:58: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceLastWeekday.java:93: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceLastWeekday.java:93: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                      ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceLastWeekday.java:93: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                                                                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\BankingScheduledPaymentRecurrenceLastWeekday.java:93: error: self-closing element not allowed
[ERROR]      * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.<br/>**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.<br/>**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.<br/>**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.<br/>**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
[ERROR]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\MetaPaginated.java:74: warning: no @param for o
[ERROR]     protected String toIndentedString(Object o) {
[ERROR]                      ^
[ERROR] C:\development\git\java-artefacts\client\src\main\java\au\org\consumerdatastandards\client\model\MetaPaginated.java:74: warning: no @return
[ERROR]     protected String toIndentedString(Object o) {
[ERROR]                      ^
[ERROR]
[ERROR] Command line was: "C:\Program Files\Java\jdk1.8.0_211\jre\..\bin\javadoc.exe" @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'C:\development\git\java-artefacts\client\target\apidocs' dir.
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :client
```

**Comment by vadkor** [comment]
Make sure that Maven uses JDK 8.

**Comment by vadkor** [comment]
Fixed in v1.3.0
